SEED-platform/seed
# 2203 dq duplicate rules

#### Any background context you want to provide?
Dq should alert user if duplicate rules are created.

#### What's this PR do?
On u added alert message if user accidentally created 2 identical rules.

#### How should this be manually tested?
Under dq page, create an identical rule to any existing rule and hit save. Expect to see error message:
<img width="1137" alt="Screen Shot 2020-09-17 at 9 31 53 PM" src="https://user-images.githubusercontent.com/49968203/93552401-a2e54380-f92d-11ea-88ed-32d8acd6323c.png">
<img width="1143" alt="Screen Shot 2020-09-17 at 9 32 06 PM" src="https://user-images.githubusercontent.com/49968203/93552433-b1cbf600-f92d-11ea-862f-fc0c3cad35bc.png">

Once user clears duplicate rule(s), save changes proceeds:
<img width="1134" alt="Screen Shot 2020-09-17 at 9 32 13 PM" src="https://user-images.githubusercontent.com/49968203/93552479-c5775c80-f92d-11ea-91a6-8b3108e165d4.png">

TaxLot view:
<img width="1140" alt="Screen Shot 2020-09-17 at 9 32 29 PM" src="https://user-images.githubusercontent.com/49968203/93552495-cc9e6a80-f92d-11ea-839a-94b9f0cdfaac.png">
<img width="1143" alt="Screen Shot 2020-09-17 at 9 32 39 PM" src="https://user-images.githubusercontent.com/49968203/93552499-cf995b00-f92d-11ea-972f-e54f13b8d70a.png">

Highlight duplicate rules:
<img width="1227" alt="Screen Shot 2020-09-24 at 1 21 03 PM" src="https://user-images.githubusercontent.com/49968203/94190134-ef1e0f80-fe68-11ea-86aa-4a49008143e6.png">


#### What are the relevant tickets?
#### Screenshots (if appropriate)
